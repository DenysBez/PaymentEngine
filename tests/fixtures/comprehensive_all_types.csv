type,client,tx,amount
deposit,1,1,100.0
deposit,2,2,200.0
withdrawal,1,3,25.0
withdrawal,2,4,50.0
dispute,1,1,
resolve,1,1,
deposit,3,5,50.0
dispute,3,5,
chargeback,3,5,
deposit,1,6,30.0
