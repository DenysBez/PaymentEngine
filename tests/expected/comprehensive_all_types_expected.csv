client,available,held,total,locked
1,105.0000,0.0000,105.0000,false
2,150.0000,0.0000,150.0000,false
3,0.0000,0.0000,0.0000,true
